llvm/test/tools/llvm-mca/RISCV/SiFiveP400/vle-vse-vlm.s

```diff
@@ -0,0 +1,542 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+# RUN: llvm-mca -mtriple=riscv64 -mcpu=sifive-p470 -iterations=1 < %s | FileCheck %s
+
+vsetvli zero, zero, e8, mf2, ta, ma
```

Comment:
Can we sort these by increasing lmul size so mf8 is before mf4 is before mf2.

Reply:
Done